tests/setup/setup.js

```diff
@@ -1,5 +1,5 @@
 import { configure } from 'enzyme';
 import AnswersAdapter from './enzymeadapter';
-import 'regenerator-runtime/runtime';
-
+import regeneratorRuntime from 'regenerator-runtime/runtime';
+global.regeneratorRuntime = regeneratorRuntime;
```

Comment:
I thought it's already defined globally with the way we import before. why do we need to update it to set it to global object this way?

Reply:
it was very strange, but we actually had different versions of regenerator-runtime in our repo EVEN with npm-force-resolutions
this was probably due to having 0.13.7 in our package.json but 0.13.1 in our "resolutions"
![image](https://user-images.githubusercontent.com/23005393/159353448-c63a52dd-a0b1-4a8e-8518-c771a3bf2fb4.png)

we were using 0.13.1 for any transient deps, but our direct dep was using 0.13.9, which is the one used here in test setup
When using npm overrides, npm will complain if you try to override a version that doesn't match up with your package.json. For example, prior to this PR we had 0.13.7 in our package.json but 0.13.1 in our "resolutions". npm didn't like that, so we're now using 0.13.1 for both our source code AND our test setup.

With regards to test setup, 0.13.1 and 0.13.9 need to be specified differently. Since the package is still v0 the maintainer was free to make breaking changes like this between 0.13.1 and 0.13.9